src/components/movie-card/movie-card.jsx

```diff
@@ -59,9 +59,13 @@ export function MovieCard({
   }, []);
 
   return (
-    <div className={colSpanOne ? "col-span-1" : colSpanClass}>
+    <div
+      className={colSpanOne ? "col-span-1" : colSpanClass}
+      data-test-id="moviecard-wrapper-div"
```

Comment:
```suggestion
      data-test-id={`moviecard-wrapper-div${card.title ? "-"+card.title : "" }`}
```
that should be good enough to make it unique for the component.